p2p/client.go

```diff
@@ -191,15 +191,32 @@ func (s *peer) getPeerID(id identity.DID) (libp2pPeer.ID, error) {
 	}
 
 	if !s.disablePeerStore {
-		// Decapsulate the /ipfs/<peerID> part from the target
-		// /ip4/<a.b.c.d>/ipfs/<peer> becomes /ip4/<a.b.c.d>
-		targetPeerAddr, _ := ma.NewMultiaddr(fmt.Sprintf("/ipfs/%s", pid))
-		targetAddr := ipfsAddr.Decapsulate(targetPeerAddr)
+		nc, err := s.config.GetConfig()
```

Comment:
fix for the KValue=20 problem

Reply:
commenting this and uncommenting the prev code below will fail the travis build because an address could not be found.